src/language-graphql/printer-graphql.js

```diff
@@ -125,10 +126,18 @@ function genericPrint(path, options, print) {
     }
     case "StringValue": {
       if (node.block) {
+        const plugins = [
+          require("../language-markdown/index.js"),
+          require("../language-graphql/index.js"),
+          require("../language-js/index.js"),
+        ];
+        const formatMarkdown = (v) => formatWithCursor(v, { plugins, parser: "markdown" });
+        const { formatted } = formatMarkdown(node.value.replace(/"""/g, "\\$&"));
```

Comment:
This is not the right way to fo call anther parser. I suggest you focus on fixing #12608.